02-Advanced-topics/05-GSD-visualization.ipynb

```diff
@@ -0,0 +1,215 @@
+{
```

Comment:
<p>use <code>fresnel.pathtrace</code> in place of <code>fresnel.preview</code>. You may need to increase the number of light samples from the default to reduce noise in the image.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/fresnel-examples/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>